### DISCUSSION TITLE
[Unstable] Destroy alien terminal achievement broken?

### DISCUSSION THREAD

**Discussion by Ek-Videogames** [FakeFishGames/Barotrauma#14685] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Destroying either variant (terminal1 type or terminal2 type) with any smallalienartifact, or alien curio specificially - doesn't trigger the achievement. Even though you said that they should be implemented on the unstable build now, so it's likely bugged somehow.

- Considering the only unstable achievement that seems to be showing up in activity is the cave mantis, its reasonable to assume that none of these other achievements are triggering right either. (such as king of the hull)

### Reproduction steps

![image](https://github.com/user-attachments/assets/45a8cfcf-b48f-4a59-b74b-4d94bad7d2eb)

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.6.4.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Ek-Videogames** [comment]
- Also just a heads up:
In previous unstable version (v1.6.10), with a small multiplayer crew on ironman mode, we made it out of the first biome with no deaths and nobody got the achievement for it.

**Comment by Jasontti** [comment]
Tested the fix for ancient novelty achievement.

Tested other achievements too again and they all unlocked properly.
Some community members have also reported that they could unlock achievements without problems.

Hard to say why this would be happening, maybe the save file got flagged for enablecheats for some reason?

**Reply by Ek-Videogames** [reply]
Already decompiled the save and there isn't anything that should be causing bugs.

Unfortunately I can't get a multiplayer crew together again to do a campaign to re-test all of the ironman achievements & also the PVP achievements, however I trust it in good faith that they're probably fixed as of that update earlier this week. (v1.6.12.0)
